### PR TITLE
chore: migrate CircleCI image

### PR DESCRIPTION
**Pull request by DevinCodes**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | no
| New feature?      | no    
| BC breaks?        | no     
| Need Doc update   | no

## Describe your change
This PR changes the Docker image used to run our tests. The current one in use will be deprecated by CircleCI.

### CODE REVIEW

**Comment by DevinCodes** [comment]
I'll update this PR once #362 has been merged.

File: .circleci/config.yml [hunk 1]
```diff
@@ -27,8 +27,8 @@ jobs:
     steps:
       - checkout
 
-      - run: sudo apt update
-      - run: sudo docker-php-ext-install zip
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y php<<parameters.version>>-zip
```
Comment (DevinCodes) [inline .circleci/config.yml:31]:
```suggestion
      - run: sudo apt-get install -y php<<parameters.version>>-zip libpq-dev php<<parameters.version>>-pgsql
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -27,8 +27,8 @@ jobs:
     steps:
       - checkout
 
-      - run: sudo apt update
-      - run: sudo docker-php-ext-install zip
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y php<<parameters.version>>-zip libpq-dev php<<parameters.version>>-pgsql
```
Comment (DevinCodes) [inline .circleci/config.yml:31]:
```suggestion
      - run: sudo apt-get install -y php<<parameters.version>>-zip libpq-dev php<<parameters.version>>-pgsql php<<parameters.version>>-mysql
```

**Review by damcou — APPROVED** [review]
👍

**Pull request merged**